pkg/azurefile/controllerserver.go

```diff
@@ -267,11 +267,24 @@ func (d *Driver) CreateVolume(ctx context.Context, req *csi.CreateVolumeRequest)
 			accountQuota = int32(value)
 		case tagValueDelimiterField:
 			tagValueDelimiter = v
+		case encryptInTransitField:
+			var err error
+			encryptInTransit, err = strconv.ParseBool(v)
+			if err != nil {
+				return nil, status.Error(codes.InvalidArgument, fmt.Sprintf("Volume context property %q must be a boolean value: %v", k, err))
+			}
 		default:
 			return nil, status.Errorf(codes.InvalidArgument, "invalid parameter %q in storage class", k)
 		}
 	}
 
+	if encryptInTransit {
```

Comment:
`enableHTTPSTrafficOnly := encryptInTransit` for nfs mount

Reply:
What `enableHTTPSTrafficOnly` is used for aznfs? Seems without this setting also working fine.